functions/index.js

```diff
@@ -99,6 +99,57 @@ const signupUser = (req, res) => {
         });
 };
 
+//Create guest account
+//funciona
+const guestAccount = (req, res) => {
+    const newGuest = {
+        username: req.body.username ||"",
+        userId: req.body.userId ||"",
+        imageProfile: req.body.imageProfile ||"",
+    };
+    if (newGuest.username == undefined) {
+        return res.status(400).json({ error: "Missing require parameter" });
+    }
+    console.log("Creating guest account..", newGuest.username);
+
+    db.doc(`/users/${newGuest.username}`)
```

Comment:
it feels strange we're storing the guest accounts in the same place as the other accounts, I think it's better if we create another collection, guest-users, and store them there instead